applications/StructuralMechanicsApplication/structural_mechanics_application.cpp

```diff
@@ -98,6 +101,12 @@ void KratosStructuralMechanicsApplication::Register()
     KRATOS_REGISTER_VARIABLE( SHELL_MOMENT )
     KRATOS_REGISTER_VARIABLE( SHELL_MOMENT_GLOBAL )
 
+
+    // Membrane1 vairiables
+    KRATOS_REGISTER_VARIABLE( PRESTRESS_11 )
```

Comment:
I would define this as a Vector variable (of variable siz,e to be able to handle both the 2D and 3D case)